scss/inc/rtl/_action-bar.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2021 Open Assessment Technologies SA
+*/
+
+.runner {
+  .test-runner-scope {
+    .action-bar.content-action-bar.horizontal-action-bar.bottom-action-bar {
+      .control-box {
+
+        & > .lft {
+          float: right;
+          margin: 2px 10px 0 0;
+        }
+        & > .rgt {
+          float: left;
+        }
+        .action {
+          float: right;
+          .icon {
+            padding: 0 0 0 9px;
+            &.icon-forward:before {
+              content: "\e68d";
```

Comment:
**nitpick:** Please use the mixin instead? Like here: https://github.com/oat-sa/tao-core-ui-fe/blob/faca56fa735b279500420c9b79147ddfeaedeb9e/scss/inc/fonts/_tao-icon-classes.scss#L166

**suggestion:** use the mixin
```suggestion
              @include icon-backward;
```